schema/article.js

```diff
@@ -22,6 +24,13 @@ const ArticleType = new GraphQLObjectType({
       type: AuthorType,
       resolve: ({ author }) => author,
     },
+    channel_id: {
+      type: GraphQLString,
+    },
+    contributing_authors: {
+      type: new GraphQLList(AuthorType),
+      resolve: ({ contributing_authors }) => contributing_authors,
+    },
```

Comment:
Yep! Because it's defined as a `GraphQLList` above it returns results like:
```
"contributing_authors": [
  {
    "id": "xxxxxx",
    "name": "Abigail"
  },
  {
    "id": "xxxxxx",
    "name": "Anna Louie"
  }
]
```